app/assets/stylesheets/layout/_utilities.scss

```diff
@@ -31,6 +31,6 @@ margin-bottom: 90px;
 
 }
 
-.btn-success {
- margin-top: 10px;
+.row {
```

Comment:
@chamyndu ".row" is already a bootstrap class, you shouldn't override it here, use an other class name (it could break the bootstrap class)